### DISCUSSION TITLE
Discord OAuth: Failed to obtain access token

### DISCUSSION THREAD

**Discussion by ruscalworld** [requarks/wiki#3420] [Help / Questions]
**Question**
I use Discord OAuth in my wiki. When user tries to sign in using Discord, it shows error "Failed to obtain access token". It worked well about a week ago, but now it doesn't work. I also have another instance of wiki.js on Ubuntu 20.04 with MariaDB and authorization works well there. I tried to make POST request as specified [here](https://discord.com/developers/docs/topics/oauth2#authorization-code-grant) (also tried discordapp.com) using CURL on my server to get access token and Discord gives me normal response. 

**Host Info (please complete the following information):**
OS: Docker
Wiki.js version: 2.4.107
Database engine: postgres 11.8

**Comment by Smankusors** [comment]
Duplicate of #2164 

Btw how do you sent the CURL request?

**Comment by ruscalworld** [comment]
I've did something like this
```
curl --data "client_id=AAA&client_secret=BBB&grant_type=authorization_code&code=CCC&redirect_uri=https://example.com/login/discord/callback&scope=identify%20email%20guilds" https://discordapp.com/api/oauth2/token
```
where  
**AAA** - client ID from discord app settings  
**BBB** - client secret from discord app settings  
**CCC** - code from GET parameters on `https://example.com/login/discord/callback`
